src/components/widgets/LabelCollapsable.vue

```diff
@@ -28,12 +30,24 @@ function handleClick() {
 
 <template>
   <div class="osim-collapsable-label" v-bind="$attrs">
-    <button type="button" class="me-2" @click="handleClick">
+    <div v-if="length === 0">
+      <slot name="label">
+        <label class="ms-2 form-label">
+          {{ `${label}: 0` }}
+        </label>
+      </slot>
+    </div>
+    <button
+      v-else
+      type="button"
+      class="me-2"
+      @click="handleClick"
+    >
       <i v-if="isExpanded" class="bi bi-dash-square-dotted me-1"></i>
       <i v-else class="bi bi-plus-square-dotted me-1"></i>
       <slot name="label">
         <label class="ms-2 form-label">
-          {{ label }}
+          {{ length > 0 ? `${label}: ${length}` : `${label}` }}
         </label>
       </slot>
     </button>
```

Comment:
My original intent with this component was to be a replacement for details/summary, I think it might not need to know anything about its contents. 

However, I think it would make sense to either:
 1. Add a prop `isExpandable` to disable the expand button (gray it out, or hide it)
 2.  Use a `disabled` fall-through attribute to gray the button. This would be my preference because it would let the user know that usually a clickable button is there, just that it can't be clicked when there are no available trackers.

Reply:
Totally agree, the length approach doesn't look clean to me now as well. 

I opted for adding `isExpandable` property on `LabelCollapsable` so false statements passed from parent components will disable the button.

IMO the fall-through limits the component's re-usability, as currently is being used only within lists, but we might want to use it in the future for non-list Labeled Collapsible.